build/scripts/VerifyExecutePermissions.ps1

```diff
@@ -0,0 +1,87 @@
+param(
+    [Parameter(Mandatory = $true)]
+    [string] $ModulesDirectory

```

Comment:
I know we only call it for the system modules now, but technically `$ModulesDirectory` is just an app project folder.
I suppose this check would be added for other apps as well, so let's adjust that.